BigBaseV2/src/core/globals.hpp

```diff
@@ -196,6 +201,8 @@ namespace big
 			this->tunables.no_idle_kick = j["tunables"]["no_idle_kick"];
 
 			this->self.godmode = j["self"]["godmode"];
+			this->self.invisibility = j["self"]["invisibility"];
+			this->self.invisibility = j["self"]["clean_player"];
```

Comment:
Assigning 2 different json values to the same global?

Set these alphabetically, to follow the same coding style as the entire project.